techniques/fileDistribution/copyGitFile/2.1/metadata.xml

```diff
@@ -14,7 +14,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<TECHNIQUE name="Download a file from the shared folder">
+<TECHNIQUE name="File download (from the policy server)">
```

Comment:
Or maybe even, judging by it's little brother "File download (HTTP)", maybe just "File download (Rudder)" ?